scidrivers.tex

```diff
@@ -24,11 +24,11 @@ \section{The LSST Science Drivers }
 \end{itemize}
 
 The SWG report\footnote{Available as
-http://www.lsst.org/Science/docs/DRM2.pdf} (also known as the Strauss
+\url{http://www.lsst.org/Science/docs/DRM2.pdf}} (also known as the Strauss
```

Comment:
@ivezic this document does not exist. Do you know where we can get it? Is there a copy on Docushare we can link to?